app-directory/.gitignore

```diff
@@ -0,0 +1,98 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
We can keep this file for now, but ideally we should have one .gitignore at the top level, not maintain multiple ones in subfolders.